test/unit/adjustTier_Unit.t.sol

```diff
@@ -1193,12 +1193,83 @@ contract Test_adjustTier_Unit is UnitTestSetup {
                 allowOwnerMint: false,
                 useReserveBeneficiaryAsDefault: false,
                 transfersPausable: false,
-                useVotingUnits: false
+                useVotingUnits: false  // <-- If false, voting power is based on tier price
             });
         }
 
-        // TODO: Shouldn't this be `.votingUnits` (and not `.category`)?
```

Comment:
useVotingUnits being false, the voting power is established with the tier price -> this setup is not needed here + added a test when using the voting unit